meson.build

```diff
@@ -166,6 +166,17 @@ if mutter42_dep.found()
 	vala_flags = ['--define', 'HAS_MUTTER338', '--define', 'HAS_MUTTER40', '--define', 'HAS_MUTTER41', '--define', 'HAS_MUTTER42']
 endif
 
+mutter44_dep = dependency('libmutter-11', version: ['>= 43', '< 45'], required: false)
```

Comment:
I'm a little confused. I suppose you were too, because GNOME releases no longer does the odd / even releases to distinguish between stable and development releases. :)

Reply:
oops, indeed, fixed